pkg/api/k8s/statehandler.go

```diff
@@ -0,0 +1,97 @@
+/*
+Copyright 2016 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package k8s
+
+import (
+	"fmt"
+
+	"github.com/rook/rook/pkg/cephmgr/mon"
+	"github.com/rook/rook/pkg/cephmgr/rgw"
+	"github.com/rook/rook/pkg/clusterd"
+	"github.com/rook/rook/pkg/model"
+	"github.com/rook/rook/pkg/operator/k8sutil"
+	"k8s.io/client-go/1.5/kubernetes"
+)
+
+type stateHandler struct {
+	clientset   *kubernetes.Clientset
+	context     *clusterd.DaemonContext
+	clusterInfo *mon.ClusterInfo
+}
+
+func New(clientset *kubernetes.Clientset, context *clusterd.DaemonContext, clusterInfo *mon.ClusterInfo) *stateHandler {
+	return &stateHandler{clientset: clientset, context: context, clusterInfo: clusterInfo}
+}
+
+func (s *stateHandler) EnableObjectStore() error {
+	logger.Infof("Enabling the object store")
+	//resource := &v1beta1.ThirdPartyResource{}
+	//_, err := s.clientset.ThirdPartyResources().Create(resource)
+	if err := createObjectUser(s.clientset, s.context, s.clusterInfo); err != nil {
+		return fmt.Errorf("failed to create the object store. %+v", err)
+	}
+
+	return nil
+}
+
+func (s *stateHandler) RemoveObjectStore() error {
+	logger.Infof("TODO: Remove the object store")
```

Comment:
i think it's consistent to simply search for FIX or TODO, no? As a log statement i just wanted to be clear that it's just not implemented at all, rather than a comment that says we'll be improving something.